master/internal/rbac/audit/audit.go

```diff
@@ -0,0 +1,34 @@
+package audit
+
+import (
+	"context"
+
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	// GetMethod is the string for the get method.
+	GetMethod = "get"
+	// PostMethod is the string for the post method.
+	PostMethod = "post"
+	// DeleteMethod is the string for the delete method.
+	DeleteMethod = "delete"
+	// PutMethod is the string for the put method.
+	PutMethod = "put"
+)
+
+// LogKey is the key used to store and extract logrus fields from context.
+type LogKey struct{}
+
+// ExtractLogFields retrieves logrus Fields from a context, if it exists.
+func ExtractLogFields(ctx context.Context) logrus.Fields {
+	fields := ctx.Value(LogKey{})
+	if fields == nil {
```

Comment:
remove this ```if fields == nil``` check I think and the ```!ok``` case should catch the nil case?

Reply:
tested and works like you said!